packages/frontend/src/app/(side-nav)/scaling/_components/scaling-filter-context.tsx

```diff
@@ -153,10 +89,10 @@ export function useScalingUpcomingFilter() {
     (entry: ScalingUpcomingEntry) => {
       const checks = [
         scalingFilters.rollupsOnly !== false
-          ? entry.category?.includes('Rollup')
+          ? !!entry.category?.includes('Rollup')
           : undefined,
-        scalingFilters.category !== undefined
-          ? entry.category === scalingFilters.category
+        scalingFilters.type !== undefined
+          ? entry.category === scalingFilters.type
```

Comment:
let's refactor this part too

Reply:
Once upcoming uses the same common base `useScalingUpcomingFilter` can be removed and replaced with `useScalingFilter`